apps/src/codebridge/hooks/useSource.ts

```diff
@@ -53,8 +56,19 @@ export const useSource = (defaultSources: ProjectSources) => {
   );
 
   const resetToStartSource = useCallback(() => {
-    setSource(levelStartSource || (defaultSources.source as MultiFileSource));
-  }, [defaultSources.source, levelStartSource, setSource]);
+    // When resetting in start mode, we always use the level start source.
+    setSource(
+      (!isStartMode && templateStartSource) ||
```

Comment:
@molly-moen Note this change in behavior when resetting. I think there's an open question around what the "initial" sources for start mode should be when resetting the workspace.

Previously, we would use the template sources, which was obviously a bug. Here, I propose using the level's (most recently saved) start sources, which matches the other labs I'm familiar with.

Reply:
It makes sense to me to reset to the most recent start sources!